x-pack/plugins/infra/public/pages/metrics/metric_detail/index.tsx

```diff
@@ -19,11 +21,25 @@ export const NodeDetail = () => {
     params: { type: nodeType, node: nodeName },
   } = useRouteMatch<{ type: InventoryItemType; node: string }>();
 
-  useMetricsBreadcrumbs([
-    {
-      text: nodeName,
-    },
-  ]);
+  const breadCrumbs: ChromeBreadcrumb[] = [];
+
+  const hostsLinkProps = useLinkProps({
+    app: 'metrics',
+    pathname: 'hosts',
+  });
+
+  if (nodeType === 'host') {
```

Comment:
I think we want to fix it for all types not only `Host` one?

@jennypavlova can you confirm that, please?

Reply:
Yes, I already added a [comment](https://github.com/elastic/kibana/pull/170792#discussion_r1386422670) about this